staging/src/k8s.io/apiserver/pkg/util/flowcontrol/exempt_borrowing_test.go

```diff
@@ -31,6 +32,9 @@ import (
 )
 
 func TestUpdateBorrowing(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
In unit tests use `_, ctx := ktesting.NewTestContext(t)` to get per-test output.

Reply:
done